tests/twind_test.ts

```diff
@@ -0,0 +1,329 @@
+import { assert, delay, puppeteer, TextLineStream } from "./deps.ts";
+
+import { cmpStringArray } from "./fixture_twind/utils/utils.ts";
+
+Deno.test({
+  name: "twind static test",
+  async fn(t) {
+    // Preparation
+    const serverProcess = Deno.run({
+      cmd: ["deno", "run", "-A", "./tests/fixture_twind/main.ts"],
+      stdout: "piped",
+      stderr: "inherit",
+    });
```

Comment:
We've just stabilized the `Deno.Command` API: https://deno.com/blog/v1.31#api-stabilizations

Can you use that instead?